src/constants/announcements.ts

```diff
@@ -1,24 +1,19 @@
+import IntroImprovedSearch from 'components/announcements/IntroImprovedSearch'
 import IntroProjectTags from 'components/announcements/IntroProjectTags'
-import IntroSepanaSearch from 'components/announcements/IntroSepanaSearch'
 import { Announcement } from 'models/announcement'
-import { NextRouter } from 'next/router'
 
 export const announcements: Announcement[] = [
```

Comment:
this is where in-app announcements are registered. currently we only have the Project Tags and Improved Search. both will expire on May 1 (arbitrary), meaning no user will see them after that date even if they hadn't been shown the announcement before.